src/genie/libs/parser/asa/show_traffic.py

```diff
@@ -0,0 +1,177 @@
+''' show_traffic.py
+
+Parser for the following show commands:
+    * show traffic
+'''
+
+# Python
+import re
+
+# Metaparser
+from genie.metaparser import MetaParser
+from genie.metaparser.util.schemaengine import Schema, Any, Optional
+
+# =============================================
+# Schema for 'show traffic'
+# =============================================
+class ShowTrafficSchema(MetaParser):
+    """Schema for
+        * show traffic
+    """
+    schema = {
+        Any(): {
+            'received': {
+                'duration': int,
+                'packets': int,
+                'bytes': int,
+                'bytes_sec': int,
+                'packets_sec': int
+            },
+            'transmitted': {
+                'duration': int,
+                'packets': int,
+                'bytes': int,
+                'bytes_sec': int,
+                'packets_sec': int                
+            },
+            'packets_input_1_minute': int,
+            'bytes_input_1_minute': int,
+            'packets_output_1_minute': int,
+            'bytes_output_1_minute': int,
+            'packets_drop_rate_1_minute': int,
+            'packets_input_5_minute': int,
+            'bytes_input_5_minute': int,
+            'packets_output_5_minute': int,
+            'bytes_output_5_minute': int,
+            'packets_drop_rate_5_minute': int
+        },
+    }
+
+# =============================================
+# Parser for 'show traffic'
+# =============================================
+class ShowTraffic(ShowTrafficSchema):
+    """Parser for
+        * show traffic
+    """
+
+    cli_command = 'show traffic'
+
+    def cli(self, output=None):
+        if output is None:
+            # excute command to get output
+            out = self.device.execute(self.cli_command)
+        else:
+            out = output
+
+        ret_dict = {}
+
+
+        #Single output from 'show traffic' command
```

Comment:
Please put examples above their respective regular expression so we know which one is which

Reply:
Corrected thank you